backend/src/layer/trigger/api/query.rs

```diff
@@ -64,6 +64,7 @@ impl QueryRoot {
 
     /// This query returns the version of this API schema. It can also be used for health checks.
     #[instrument(skip(self, _ctx))]
+    #[allow(clippy::no_effect_underscore_binding)]
```

Comment:
about what is clippy complaining here? whey do we ignore that?
https://rust-lang.github.io/rust-clippy/master/index.html#/underscore_b

Reply:
alright, I think this is a problem with the linter not detecting that this argument is required because of the macro. Anoying but ignoring is right.